src/material/form-field/form-field.md

```diff
@@ -27,7 +27,7 @@ There are a couple differences to be aware of between the `legacy` appearance an
 `standard`, `fill`, and `outline` appearances. The `matPrefix` and `matSuffix` elements are center
 aligned by default for the newer appearances. The Material Design spec shows this as being the
 standard way to align prefix and suffix icons in the newer appearance variants. We do not recommend
-using text prefix and suffixes in the new variants because the label and input do not have the same
+using text prefix and suffixes in the 'fill' and 'outline' variants because the label and input do not have the same
```

Comment:
nit: the variant names should be wrapped with backticks, instead of quotes, so they should as code (e.g. `fill` vs 'fill').

Reply:
I've updated the PR